js/w3c/style.js

```diff
@@ -73,15 +73,16 @@ define(
             case "LC-NOTE":
               styleFile += "WD";
               break;
+            case "WG-NOTE":
             case "FPWD-NOTE":
-              styleFile += "WG-NOTE";
+              styleFile += "WG-NOTE.css";
```

Comment:
How odd? This is what I get?

![screenshot 2016-04-20 12 47 02](https://cloud.githubusercontent.com/assets/870154/14662046/0e0bbb8c-06f6-11e6-9950-8064c37f067c.png)

Why am I seeing something different to you?